qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -85,6 +85,9 @@
                     only pseries
                     target_hugepages = 266
                     expected_hugepage_size = 16384
+                    only RHEL_ALT
+                    target_hugepages = 2058
+                    expected_hugepage_size = 2048
```

Comment:
better to handle in the code with target supported hugepage sizes vs given and change proceed/cancel test...

Reply:
Hi sathnaga,
    Thanks for your comments.In this case,there are different huge page size for different architectures.A cfg maintains them much more flexible,in my opinions.